modules/phase_field/doc/content/source/kernels/ACInterfaceCleavageFracture.md

```diff
@@ -0,0 +1,48 @@
+# ACInterfaceCleavageFracture
+
+!syntax description /Kernels/ACInterfaceCleavageFracture
+
+Implements the Allen-Cahn term with crack propagation preferred along weak
+cleavage plane specified using cleavage plane normal ($\boldsymbol{n}$). Planes
+not normal to $\boldsymbol{n}$ are penalized using $\beta$ parameter
+[!cite](ClaytonKnap2015). Setting $\beta=0$ results in isotropic damage with
+respect to cleavage planes. The added term is
+
+\begin{equation}
+\dfrac{\beta l_0}{2} \left( \boldsymbol{I} - \boldsymbol{n} \otimes \boldsymbol{n} \right) : \left( \nabla d \otimes \nabla d \right)
+\end{equation}
+
+Its weak form is
+
+\begin{equation}
+\int_{\Omega} \left( - \beta l_0 \left( \nabla^2 d - \boldsymbol{n} \otimes \boldsymbol{n} : \nabla \nabla d \right) \right) \psi dV
+\end{equation}
+
+The second term in above expression can be simplified as,
+
+\begin{equation}
+\begin{aligned}
+\int_{\Omega} \psi \left(\boldsymbol{n} \otimes \boldsymbol{n} : \nabla \nabla d \right) dV
+&= \int_{\Omega} \left( \nabla \cdot \left( \psi \boldsymbol{n} \left( \boldsymbol{n} \cdot \nabla d \right) \right) - \left( \boldsymbol{n} \cdot \nabla\psi \right) \left( \boldsymbol{n} \cdot \nabla d \right) - \left( \psi \nabla \cdot \boldsymbol{n} \right) \left( \boldsymbol{n} \cdot \nabla d \right) \right) dV
+\
+&= \int_{\Omega} \nabla \cdot \left( \psi \boldsymbol{n} \left( \boldsymbol{n} \cdot \nabla d \right) \right) dV - \int_{\Omega} \left( \boldsymbol{n} \cdot \nabla \psi \right) \left( \boldsymbol{n} \cdot \nabla d \right) dV - \int_{\Omega} \left( \psi \nabla \cdot \boldsymbol{n} \right) \left( \boldsymbol{n} \cdot \nabla d \right) dV
```

Comment:
The notation \boldsymbol{n} is a bit confusing. Can you use M for cleavage normal and n for the boundary normal?

And you might want to replace `\` with `\\` to put equations on multiple lines.

Reply:
Thanks for the comment. I changed the notation. Now it is, M for cleavage plane normal and n for boundary normal. Also, the multiline equation now uses \\.